bucket/staxrip.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "2.6.0",
+    "description": "Video encoding GUI for Windows.",
+    "homepage": "https://github.com/staxrip/staxrip",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/staxrip/staxrip/releases/download/v2.6.0/StaxRip-v2.6.0-x64.7z",
+            "hash": "78730a6554507e8512a7537a689fafd595d316e796249a7ee5b9577a7ab03fce",
+            "extract_dir": "StaxRip-v2.6.0-x64"
+        }
+    },
+    "shortcuts": [
+        [
+            "StaxRip.exe",
+            "StaxRip"
+        ],
+        [
+            "Apps\\Support\\SubtitleEdit\\SubtitleEdit.exe",
+            "StaxRip Apps\\Subtitle Edit"
+        ],
+        [
+            "Apps\\Support\\MKVToolNix\\mkvtoolnix-gui.exe",
+            "StaxRip Apps\\MKVToolnix GUI"
+        ],
+        [
+            "Apps\\Support\\chapterEditor\\chapterEditor.exe",
+            "StaxRip Apps\\chapterEditor"
+        ]
+    ],
+    "persist": [
+        "Apps",
```

Comment:
Persisting folder, which is boundled by application and updated on every release is useless.

Reply:
Not all the apps are bundled actually. Apple Support Library and libfdkaac, for example, are not included in the archive for license issues.